docs/snippets/all/views/spatial2d.py

```diff
@@ -7,21 +7,25 @@
 rr.init("rerun_example_spatial_2d", spawn=True)
 
 # Create a spiral of points:
-theta = np.linspace(0, 10 * np.pi, 300)
-radius = np.linspace(0, 10, 300)
-positions = np.column_stack((np.cos(theta) * radius, np.sin(theta) * radius))
-colors = np.random.randint(0, 255, size=(len(theta), 3))
+n = 150
+angle = np.linspace(0, 10 * np.pi, n)
+spiral_radius = np.linspace(0.0, 3.0, n) ** 2
+positions = np.column_stack((np.cos(angle) * spiral_radius, np.sin(angle) * spiral_radius))
+r = np.linspace(255, 255, n).astype(int)
+g = np.linspace(255, 0, n).astype(int)
+b = np.linspace(0, 255, n).astype(int)
+colors = np.dstack((r, g, b))[0]
+radii = np.linspace(0.01, 0.7, n)
```

Comment:
I asked chatgpt to make this shorter, the only useful thing it came up with is to have  only a single `astype(int)` call:

```suggestion
n = 150
angle = np.linspace(0, 10 * np.pi, n)
spiral_radius = np.linspace(0.0, 3.0, n) ** 2
positions = np.column_stack((np.cos(angle) * spiral_radius, np.sin(angle) * spiral_radius))
r = np.linspace(255, 255, n)
g = np.linspace(255, 0, n)
b = np.linspace(0, 255, n)
colors = np.dstack((r, g, b))[0].astype(int)
radii = np.linspace(0.01, 0.7, n)
```
at which point inlining r/g/b is maybe ok with our formatter?